sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -51,3 +51,97 @@ context-client-method-parameter: true
 enable-xml: false
 required-parameter-client-methods: true
 ```
+### Rename AddChatThreadMembersRequest to AddChatThreadMembersOptions
```

Comment:
Good job figuring that out!

(I'm assuming you validated by regenerating 🙃)

Reply:
haha thanks, and yes I have regenerated and validated that files are renamed correctly!